STL_Extension/include/CGAL/STL_Extension/internal/mesh_parameters_interface.h

```diff
@@ -55,7 +55,7 @@ perturb(const CGAL_NP_CLASS& np = parameters::default_values())
 }
 
 template<typename CGAL_NP_TEMPLATE_PARAMETERS_NO_DEFAULT_1, typename CGAL_NP_TEMPLATE_PARAMETERS_NO_DEFAULT_2, typename ... NP>
-Named_function_parameters<::CGAL::parameters::internal::Exude_options, ::CGAL::internal_np::exude_options_param_t, CGAL_NP_BASE>
+Named_function_parameters<::CGAL::parameters::internal::Perturb_options, ::CGAL::internal_np::perturb_options_param_t, CGAL_NP_BASE>
```

Comment:
Do we know why the previous code compiled anyway?

Seeing the patch, it seems it was completely broken.

Reply:
IIRC it didn't compile, it just wasn't tested.

Reply:
![image](https://github.com/CGAL/cgal/assets/5746675/56bcc6f5-762f-4710-847c-51f2f51f4060)

I wonder why that "face palm" emoji is not yet an official one.

Reply:
:facepalm:

Reply:
the parameters names are almost all the same (angle_bound, max_time, etc) so it may even compile depending on which parameters are used

Reply:
Actually, `perturb(NP&)` is tested. The overload that was not tested was only for the backward-compatibility with the version of Mesh_3 with Boost Parameters.

And "If it isn't tested, it's broken".

Reply:
- `perturb(const CGAL_NP_CLASS&)` is tested
- `perturb(const CGAL_NP_CLASS_1&  np1, const CGAL_NP_CLASS_2&  np2, const NP& ... nps)` was not, and is not. It is just "tested" in the benchmark code.